models/ride_models.py

```diff
@@ -18,10 +20,9 @@ def __init__ (self,location,destination):
 
 class DriverOffer:
     """Driver offers ride"""
```

Comment:
Most of your classes don't have docstrings but this class does. Don't you think it would be best if you had consistency in your code?